.gitlab-ci.yml

```diff
@@ -339,18 +339,21 @@ libvirt:
           - openstack/rhel-9.0-beta-nightly-x86_64
         INTERNAL_NETWORK: ["true"]
 
-RHEL 9 on 8:
```

Comment:
RHEL 9 on 8 is still needed. RHEL 9 guest images and ec2 images are still built on RHEL 8.

We don't need to test this on nightlies though, this is not officially supported.